raiden/ui/cli.py

```diff
@@ -538,3 +541,45 @@ def append_report(subject, data):
     else:
         print('[5/5] smoketest had errors, report was written to {}'.format(report_file))
         sys.exit(1)
+
+
+@run.command()
+@click.pass_context
```

Comment:
Just FYI: you could directly use the [ `@pass_obj`](http://click.pocoo.org/6/api/#click.pass_obj) decorator if all you need from the context is `obj`. But this is fine too, no need to change it.